tests/unittests/CoreGraphics.drawing/CGTextDrawing.cpp

```diff
@@ -0,0 +1,58 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+#include <CoreText/CoreText.h>
+#include <Starboard/SmartTypes.h>
+
+void __CreateGlyphs(CGGlyph glyphs[4]) {
```

Comment:
The following seem like possible improvements:

- return std::vector<CGGlyph> instead of through an out-array, removes burden of knowing size
- add input parameter of std::initializer_list<UniChar>, instead of restricting to just this set of chars